app/views/cardeditor.html

```diff
@@ -75,11 +71,49 @@ <h3>
       <i>Remove CallerDojo Logo</i>
     </button>
   </li>
+  <li>
+    <!-- <a class="twitter-button twitter-pro twitter-hashtags-button" id="buttons" href="https://twitter.com/intent/tweet" data-text="Welcome to CallerDojo"> -->
+      <a class="twitter-share-button"
+  href="https://twitter.com/intent/tweet"
+  data-size="large"
+  data-text="Look like a pro on your zoom and team calls."
+  data-url="www.callerdojo.com"
+  data-hashtags="zoom,virtual-call"
+  data-via="callerdojo"
+  data-related="twitterapi,twitter">
+Tweet
+</a><br />
+<i>Tweet to unlock PRO</i>
+  </li>
   <li>
     <button type="button" ng-click="DownloadPicture()" class="button muted button-download-file js-track-picture">
       <i class="fa fa-arrow-down"></i>
       Download
     </button>
   </li>
-
 </ul>
+
+<script>
+  function reward_user( event ) {
```

Comment:
You'll need to get this into main.js so that you can update the local storage that pro has been unlocked.

Reply:
yes that pro unlocks when you click on the Tweet button, if i take that code to the main.js, the whole code breaks, so i left it in the HTML file, but i guess its working fine

Reply:
so i figured out the issue, on local its not going to unlock because http://localhost:9000/ is not on Stripe's dashboard kindly refer to the screenshot below
<img width="1440" alt="Screenshot 2020-10-19 at 15 32 17" src="https://user-images.githubusercontent.com/29440506/96464979-5b015700-1220-11eb-8652-edaf73f3ec27.png">